### PR TITLE
ENTESB-12106 Remove the deployIntegrations flag from the CR

### PR DESCRIPTION
**Pull request by claudio4j**
https://issues.redhat.com/browse/ENTESB-12106

* Increase apiVersion to v1beta3 due to removing a field from CRD
* Remove unused `install/operator/version/version.go`
* Added a `synapi` prefix to syndesis api imports to make it easier to change api version in the future.

### CODE REVIEW

**Review by zregvart — APPROVED** [review]
LGTM

File: install/operator/config/Makefile [hunk 1]
```diff
@@ -15,7 +15,7 @@ KOPTIONS := --load-restrictor LoadRestrictionsNone
 #
 # Vars that can be overridden by external env vars
 #
-VERSION ?= 1.12.0
+VERSION ?= 1.13.0
```
Comment (phantomjinx) [inline install/operator/config/Makefile]: This is not required if https://github.com/syndesisio/syndesis/pull/9710 is merged first.

File: install/operator/config/PROJECT [hunk 2]
```diff
@@ -5,7 +5,7 @@ repo: github.com/syndesisio/syndesis
 resources:
 - group: syndesis.io
   kind: Syndesis
-  version: v1beta2
-version: "3"
+  version: v1beta3
+version: "4"
```
Comment (phantomjinx) [inline install/operator/config/PROJECT]: Why does the version need changing here? This is related to the operator-sdk rather the syndesis

File: install/operator/config/dev/operator/Makefile [hunk 3]
```diff
@@ -7,7 +7,7 @@ PATCH_ANNOTATION := ./annotation-patch
 PATCH_IMG_STREAM := ./image-stream-patch
 
 # User customisable variables
-TAG ?= 1.12
+TAG ?= 1.13
```
Comment (phantomjinx) [inline install/operator/config/dev/operator/Makefile]: Taken care of in https://github.com/syndesisio/syndesis/pull/9710

File: install/operator/config/manager/Makefile [hunk 4]
```diff
@@ -15,7 +15,7 @@ MANAGER_INSTALL := ./manager-install.gen
 DB_IMAGE ?= centos/postgresql-10-centos7
 IMAGE ?= quay.io/syndesis/syndesis-operator
 NAMESPACE ?= syndesis
-TAG ?= 1.12
+TAG ?= 1.13
```
Comment (phantomjinx) [inline install/operator/config/manager/Makefile]: Taken care of in #9710

File: install/operator/config/manifests/bases/syndesis.clusterserviceversion.yaml [hunk 5]
```diff
@@ -5,7 +5,7 @@ metadata:
     capabilities: Seamless Upgrades
     categories: Integration & Delivery
     certified: "false"
-    containerImage: quay.io/syndesis/syndesis-operator:1.12
+    containerImage: quay.io/syndesis/syndesis-operator:1.13
```
Comment (phantomjinx) [inline install/operator/config/manifests/bases/syndesis.clusterserviceversion.yaml]: Taken care of in #9710

File: install/operator/config/manifests/bases/syndesis.clusterserviceversion.yaml [hunk 6]
```diff
@@ -101,5 +101,5 @@ spec:
   provider:
     name: Syndesis Team
     url: https://github.com/syndesisio/syndesis
-  replaces: syndesis-operator.1.11.0
-  version: 1.12.0
+  replaces: syndesis-operator.1.12.0
+  version: 1.13.0
```
Comment (phantomjinx) [inline install/operator/config/manifests/bases/syndesis.clusterserviceversion.yaml]: Taken care of in #9710

File: install/operator/pkg/cmd/internal/install/install_forge.go [hunk 7]
```diff
@@ -26,7 +26,7 @@ import (
 	"unicode"
 
 	"github.com/spf13/cast"
```
Comment (phantomjinx) [inline install/operator/pkg/cmd/internal/install/install_forge.go:28]: General note: really want to remove install-forge and olm gen from syndesis binary. Neither are used.

File: install/operator/pkg/syndesis/olm/csv.go [hunk 8]
```diff
@@ -331,7 +331,7 @@ func (c *csv) build() (err error) {
 			Customresourcedefinitions: CustomResourceDefinitions{
```
Comment (phantomjinx) [inline install/operator/pkg/syndesis/olm/csv.go:331]: General note: Definitely want to stop maintaining this olm code. So out of date!

**Review by phantomjinx — CHANGES_REQUESTED** [review]
Could you just leave out the bundle changes from #9710 as I have put in a different way of maintaining those constants. Couple of general notes for us to clean up. Otherwise good to go.

File: install/operator/config/Makefile (hunk 1 above)
Comment (claudio4j) [reply install/operator/config/Makefile]: Rebased and fixed.

File: install/operator/pkg/cmd/internal/install/install_forge.go (hunk 7 above)
Comment (claudio4j) [reply install/operator/pkg/cmd/internal/install/install_forge.go:28]: Good, I will do this in another PR.

File: install/operator/pkg/syndesis/olm/csv.go (hunk 8 above)
Comment (claudio4j) [reply install/operator/pkg/syndesis/olm/csv.go:331]: Good, I will do this in another PR.

File: install/operator/config/PROJECT (hunk 2 above)
Comment (claudio4j) [reply install/operator/config/PROJECT]: Reverted the version increase.

**Review by phantomjinx — COMMENTED** [review]
LGTM

**Review by phantomjinx — APPROVED** [review]

**Pull request merged**